src/table-data-source.ts

```diff
@@ -26,12 +28,14 @@ export class TableDataSource<T> extends DataSource<TableElement<T>> {
   constructor(
     data: T[],
     dataType?: new () => T,
-    private validatorService?: ValidatorService,
+    validatorService?: ValidatorService,
```

Comment:
Maybe it could be a better approach to do the following:
`private validatorService: ValidatorService = new DefaultValidatorService(),`

Then we avoid to declare it twice. What do you think?

Reply:
Totally agree; your suggestion is better (Y)